help/set_ram.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Set maximum RAM"
+permalink: /help/set-ram/
+---
+(Valid for TripleA v1.9.0.0.5401+)<br>
+It may happen that your TripleA client exits with an `OutOfMemoryError`, even if your PC has more RAM to offer.<br>
+In this case you might want to increase the usable amount of RAM for your installation of TripleA.<br>
+There are multiple ways to achieve this, and they all depend on the way your version of TripleA has been installed on your computer.
+
+<br>
+`<amount>` is the amount of memory you want to assign to TripleA.<br>
+`1` means 1 byte, `1K` 1024 bytes, `1M` 1024 kilobytes, `1G` 1024 megabytes and so on.<br>
+The default value TripleA uses is `1G`.<br><br>
+
+ - Installer
+   - Temporary:
```

Comment:
I'm not sure there is a use-case for a temporary update, we can probably scatch the CLI steps in favor of just having the permanent fix.